overlord/overlord.go

```diff
@@ -114,12 +121,7 @@ func New() (*Overlord, error) {
 	o.ifaceMgr = ifaceMgr
 	o.stateEng.AddManager(o.ifaceMgr)
 
-	hookMgr, err := hookstate.Manager(s)
-	if err != nil {
-		return nil, err
-	}
-	o.hookMgr = hookMgr
-	o.stateEng.AddManager(o.hookMgr)
+	ifacestate.SetupHooks(hookMgr)
 
 	configMgr, err := configstate.Manager(s, hookMgr)
```

Comment:
Let's please use the same convention we've used here to pass the hook manager into the interface manager.